ci/deploy/call_webhook.sh

```diff
@@ -10,6 +10,13 @@ if [[ "$1" = "" ]]; then
   exit 1
 fi
 
-curl "$DEPLOY_STATUS_WEBHOOK" -H 'content-type: application/json' \
-    --data "`echo "$1" | jq --raw-input '{content: .}'`" \
+msg=""
+
+while [[ "$1" != "" ]]; do
```

Comment:
I am not good in Shell scripts. Please could you make this part more readable? Why do we concat things into `msg`? Maybe creating a function or adding a comment would be a solution

Reply:
At some point I thought that I want pass a long message here. To make that not a single long ugly line in callers, I want to be able to write like so:
```sh
./call_webhook.sh \
  "This is a long message, " \
  "so I've split it to several lines"
```
Added a comment, too.